docs/partials/readmeAddlInfo.md

```diff
@@ -1 +1,2 @@
-<!-- AURO-GENERATED-CONTENT This file is to be used for any additional content that should be included in the README.md which is specific to this component. -->
+**Important Note:** `<auro-accordion>` v2.0 is a complete rewrite of the component, meaning that it is **not** intended to be backwards compatible. If you would like to use an older version of `<auro-accordion>`, you will need to install `@alaskaairux/auro-accordion`.
```

Comment:
**Important Note:** `@aurodesignsystem/auro-accordion` includes breaking changes and is not backwards compatible with earlier releases.

Earlier versions may be installed by running the following command.

```shell
$ npm i @alaskaairux/auro-accordion
```

Or, by CDN.

```html
<link rel="stylesheet" href="https://unpkg.com/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
<link rel="stylesheet" href="https://unpkg.com/@alaskaairux/webcorestylesheets@latest/dist/bundled/essentials.css" />

<script src="https://unpkg.com/@alaskaairux/auro-accordion@latest/dist/auro-accordion__bundled.es5.js" nomodule></script>
<script src="https://unpkg.com/@alaskaairux/auro-accordion@latest/dist/auro-accordion-group__bundled.es5.js" nomodule></script>

<script src="https://unpkg.com/@alaskaairux/auro-accordion@latest/dist/auro-accordion__bundled.js" type="module"></script>
<script src="https://unpkg.com/@alaskaairux/auro-accordion@latest/dist/auro-accordion-group__bundled.js" type="module"></script>
```